assets/css/admin.scss

```diff
@@ -610,54 +610,76 @@ a.wpjm-activate-license-link:active {
 
 /* Addons */
 .wp_job_manager_add_ons_wrap {
+	max-width: 1244px;
 
 	.products {
-		overflow: hidden;
+		display: flex;
```

Comment:
I think we could stretch the cards to fill the container when breaking to one or two columns on smaller screens:

<table>
<tr>
 <td> 
Before
<img width="747" alt="image" src="https://github.com/Automattic/WP-Job-Manager/assets/176949/eb19d840-1e23-4cee-82d8-e765dae52455">
 <td> 
After
<img width="732" alt="image" src="https://github.com/Automattic/WP-Job-Manager/assets/176949/ace7f165-1489-4199-a0c2-71607a3acdbe">

</table>

Best option for that is grid like this:


```suggestion
		display: grid;
		grid-template-columns: repeat(auto-fit, minmax(min(100%, 400px), 1fr));
```

(Also need to remove the `width` from the `li` items for it to work.)

Reply:
Good idea!  Done in ee945efb343f39be7d7f3a85fd09bf851db9c511